hack/jenkins/e2e-runner.sh

```diff
@@ -21,6 +21,11 @@ set -o nounset
 set -o pipefail
 set -o xtrace
 
+export HOME=${WORKSPACE} # Nothing should want Jenkins $HOME
+
+# TODO(ihmccreery): Why is this here?
```

Comment:
because otherwise `go` isn't in the path, and we need `go` for running hack/e2e.go.